Added Postgres ilike method for case-insensitive like comparison

gnat42 opened the pull request:
Adds the `ILIKE(c.field, :value) = true` support

gnat42 committed:
added ilike

Chris53897 commented:
@gnat42 Please add it in the readme.md too
Like https://github.com/Dukecity/DoctrineExtensions/pull/11/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R19

iampedropiedade commented:
Hi! Is this going to be merged? Package seems a bit stale currently...

gnat42 commented:
I've added the line to the documents, and rebased on latest master branch.. I worked to clean the psalm and phpcs problems. No idea why the 8.4 tests fail in that way but not sure I changed anything that would affect that...
